apps/monitoring/src/ui/datadog/datadog-config.tsx

```diff
@@ -175,114 +143,111 @@ export const DatadogConfig = () => {
 
-            if (updatedConfig) {
-              setValue("active", updatedConfig.active);
-              setValue("apiKey", buildMaskedKey(updatedConfig.credentials.apiKeyLast4));
-              setValue("site", updatedConfig.credentials.site);
+              if (updatedConfig) {
+                setValue("active", updatedConfig.active);
+                setValue("apiKey", buildMaskedKey(updatedConfig.credentials.apiKeyLast4));
+                setValue("site", updatedConfig.credentials.site);
 
-              notifySuccess("Configuration updated", "Successfully updated Datadog settings");
-            }
+                notifySuccess("Configuration updated", "Successfully updated Datadog settings");
+              }
 
-            if (errors?.length) {
-              notifyError("Error configuring Datadog", errors[0].message);
-            }
-          });
-        })}
-      >
-        <div>
-          <Controller
-            render={({ field }) => {
-              return (
-                <FormGroup row>
-                  <FormControlLabel
-                    control={<Checkbox checked={field.value} {...field} />}
-                    label="Enabled"
-                  />
-                </FormGroup>
-              );
-            }}
-            name="active"
-            control={control}
-          />
-          <InputLabel id="datadog-site-label">Datadog Site</InputLabel>
-          <Select
-            defaultValue={DatadogSite.Us1}
-            fullWidth
-            labelId="datadog-site-label"
-            {...register("site")}
-          >
-            {Object.values(DatadogSite).map((v) => (
-              <MenuItem value={v} key={v}>
-                {v}
-              </MenuItem>
-            ))}
-          </Select>
-        </div>
-        <TextField
-          fullWidth
-          variant="standard"
-          label="Api Key"
-          defaultValue=""
-          helperText={<ApiKeyHelperText site={activeSite} />}
-          {...register("apiKey")}
-        />
-        {queryData.data?.integrations.datadog?.error && (
-          <Typography color="error">{queryData.data?.integrations.datadog?.error}</Typography>
-        )}
-        <Button type="submit" variant="primary" fullWidth>
-          Save configuration
-        </Button>
-        <Button
-          type="reset"
-          variant="secondary"
-          fullWidth
-          onClick={(e) => {
-            e.preventDefault();
-
-            deleteCredentials({}).then(() => {
-              fetchConfig();
-              reset();
-              notifySuccess("Configuration updated", "Successfully deleted Datadog settings");
+              if (errors?.length) {
+                notifyError("Error configuring Datadog", errors[0].message);
+              }
             });
-          }}
+          })}
         >
-          Delete configuration
-        </Button>
-      </form>
-    </Section>
+          <Box as={"label"} display={"flex"} gap={2} marginBottom={4}>
```

Comment:
```suggestion
          <Box as={"label"} display={"flex"} gap={2}>
```

Wrap whole form in Box with flex and gap property - cleaner than manually setting gaps in every field.